security/security-design/fn-datasafe-dbaudit-to-oci-logging/README.md

```diff
@@ -0,0 +1,146 @@
+
+2. If you aren't already signed in, when prompted, enter the tenancy and user credentials.
+
+3. Review and accept the terms and conditions.
+
+4. Select the region where you want to deploy the stack.
+
+5. Follow the on-screen prompts and instructions to create the stack.
+
+6. After creating the stack, click **Terraform Actions**, and select **Plan**.
+
+7. Wait for the job to be completed, and review the plan.
+
+8. To make any changes, return to the Stack Details page, click **Edit Stack**, and make the required changes. Then, run the **Plan** action again.
+
+9. If no further changes are necessary, return to the Stack Details page, click **Terraform Actions**, and select **Apply**.
+
+## Deploy Using the Terraform CLI
+
+### Clone the Module
+Now, you'll want a local copy of this repo. You can make that with the commands:
+
+    git clone https://github.com/oracle-devrel/technology-engineering.git
+    cd security/security-design/fn-datasafe-dbaudit-to-oci-logging
+    ls
+
+### Prerequisites
+First off, you'll need to do some pre-deploy setup for Docker and Fn Project inside your machine:
+
+```
+sudo su -
+yum update
+yum install yum-utils
+yum-config-manager --enable *addons
+yum install docker-engine
+groupadd docker
+service docker restart
+usermod -a -G docker opc
+chmod 666 /var/run/docker.sock
+exit
+curl -LSs https://raw.githubusercontent.com/fnproject/cli/master/install | sh
+exit
+```
+
+OR
+
+you'll use [Oracle Linux Cloud Developer Image](https://docs.oracle.com/en-us/iaas/oracle-linux/developer/index.htm). The Oracle Linux Cloud Developer image provides the latest development tools, languages, and Oracle Cloud Infrastructure Software Development Kits (SDKs) to rapidly deploy, that include Podman instead of Docker.
+The Oracle Linux Cloud Developer image don't include Fn Project but it easy to setup:
+
+```
+curl -LSs https://raw.githubusercontent.com/fnproject/cli/master/install | sh
+```
+
+Also, please follow this [note](https://docs.oracle.com/en-us/iaas/Content/Functions/Tasks/functionsinstalldocker.htm#Install_Docker_for_Use_with_Oracle_Functions__section_podman_instead_of_docker). By default, Fn Project (and by extension, OCI Functions) assumes the use of Docker to build and deploy function images. However, Fn Project also supports Podman as an alternative to Docker. When using Fn Project CLI version 0.6.12 and above, you can set a configuration setting to specify that you want to use Podman instead of Docker. 
+
+
+### Set Up and Configure Terraform
+
+1. Complete the prerequisites described [here](https://github.com/cloud-partners/oci-prerequisites).   
+
+2. Create a `terraform.tfvars` file, and specify the following variables:
+

```

Comment:
Add a note that this file will include sensitive information and needs to be protected from unauthorized usage (for example in CI/CI environments or on local machines).

Reply:
Added note